flyteadmin/pkg/repositories/transformers/friendly_name.go

```diff
@@ -0,0 +1,17 @@
+package transformers
+
+import (
+	"github.com/wolfeidau/humanhash"
+	"k8s.io/apimachinery/pkg/util/rand"
+)
+
+const RandStringLength = 20
+
+/* #nosec */
```

Comment:
why 20?

Reply:
It can be any number, do I need to add a comment?

Reply:
yes please, Kevin will help this one